contracts/protocol/facets/ExchangeHandlerFacet.sol

```diff
@@ -995,9 +995,6 @@ contract ExchangeHandlerFacet is IBosonExchangeHandler, BuyerBase, DisputeBase {
                 // Increment number of commits to the group for this address if they are allowed to commit
                 lookups.conditionalCommitsByAddress[_buyer][_groupId] = ++commitCount;
             }
-        } else {
-            // No condition set, so allow the commit
-            allow = true;
         }
```

Comment:
Now that you removed this line, preminted offers for groups with the condition with EvaluationMethod `None` are not possible anymore.

Maybe it looks like an edge case, but I think it's a valid one and we should support preminted offers with None condition.

For example, a seller could create a preminted offer with a group that has a threshold condition. But later they decide to remove that condition and they can do it by calling `setGroupCondition` and setting it to None. We want that preminted offers still work after that.

Reply:
actually this is not possible since would revert on line 200
https://github.com/bosonprotocol/boson-protocol-contracts/pull/635/files#diff-9734ea571f47efbe33f29fc8ab6a117aa407a1a4b8bdbb0a40771e9b0afad824R200

Reply:
You added that "require" after my review. But that does not make it any better.

I'm saying that `commitToPreMintedOffer` should succeed if `condition.method == EvaluationMethod.None` and now it doesn't.

Also now preminted offers cannot be gated with `Multitoken` which again seems a very arbitrary restriction. 
I understand that preminted offers cannot work with conditions where buyers must communicate which `tokenId` is being used to commit, but all other cases should be supported.

Reply:
> I'm saying that commitToPreMintedOffer should succeed if condition.method == EvaluationMethod.None and now it doesn't.

You're right sorry